flight/Modules/OnScreenDisplay/onscreendisplay.c

```diff
@@ -1061,6 +1062,95 @@ void showVideoType(int16_t x, int16_t y)
 	}
 }
 
+// XXX right now looks at command and does not consider at all
+// "what's happening" -- should weight both [with filtering]
+// XXX for now assumes reprojection-- should inverse-reproject if no reproj
+// XXX assumes angles are small and doesn't integrate the 3D rotation
+// properly
+static void plot_trend_centermark(int max_pitch)
+{
+#define INNER_GAP 2
+#define INNER_MARK_LEN 2
+#define OUTER_GAP 4
+#define OUTER_MARK_LEN 3
+
+#define TREND_FORECAST_LEN 0.3f
+
+	write_line_outlined(GRAPHICS_X_MIDDLE - INNER_GAP - INNER_MARK_LEN,
```

Comment:
Yeah, we discussed that earlier on IRC. This is a quick proof-of-concept opened to get build artifacts from Jenkins. ;-)